near/near-hat
# feat: add optional CLI argument for spooning contracts

To test:
```
// 17208628f84f5d6ad33f0da3bbbeb27ffcb398eac501a31bd6ad2 is USDC contract id
RUST_BACKTRACE=1 RUST_LOG=debug cargo run -p near-hat-cli -- start --contracts-to-spoon 17208628f84f5d6ad33f0da3bbbeb27ffcb398eac501a31bd6ad2011e36133a1
```

Then do:
```
curl -X POST -d '{                                                                                   
      "jsonrpc": "2.0",
      "id": "dontcare",
      "method": "query",
      "params": {
        "request_type": "call_function",
        "finality": "final",
        "account_id": "17208628f84f5d6ad33f0da3bbbeb27ffcb398eac501a31bd6ad2011e36133a1",
        "method_name": "ft_total_supply",
        "args_base64": ""
      }
    }' -H 'Content-type: application/json' <RPC_URL>
```

It should return something like:
```json
{"jsonrpc":"2.0","result":{"block_hash":"7aba9T9sk1DPxJPaTNXetRsH5JhrqHjJCDwZbB4Nz4Qx","block_height":140,"logs":[],"result":[34,52,51,49,49,54,56,55,51,51,48,48,48,48,34]},"id":"dontcare"}⏎
```
